vernac/vernacexpr.ml

```diff
@@ -121,12 +121,17 @@ type 'a hint_info_gen = 'a Typeclasses.hint_info_gen =
 type hint_info_expr = Hints.hint_info_expr
 [@@ocaml.deprecated "Please use [Hints.hint_info_expr]"]
 
+type 'a hints_transparency_target = 'a Hints.hints_transparency_target =
+  | HintsVariables
+  | HintsConstants
+  | HintsReferences of 'a list
+
```

Comment:
@ppedrot Does this alias seems totally pointless? I'd rather remove it as it has not been in a release yet.

Reply:
No idea, do as you judge good.